### PR TITLE
Fixe some typos

### PR DESCRIPTION
**Pull request by apivovarov**
### Description:
- Fixes for typos in comments

### CODE REVIEW

**Review by mikaylagawarecki — APPROVED** [review]
Thanks!

File: torch/nn/modules/activation.py [hunk 1]
```diff
@@ -886,19 +886,13 @@ def extra_repr(self) -> str:
 def _check_arg_device(x: Optional[torch.Tensor]) -> bool:
     if x is None:
         return True
-    else:
-        return x.device.type in ["cpu", "cuda", torch.utils.backend_registration._privateuse1_backend_name]
-
-    return False
+    return x.device.type in ["cpu", "cuda", torch.utils.backend_registration._privateuse1_backend_name]
```
Comment (mikaylagawarecki) [inline torch/nn/modules/activation.py:889]: Actually, it seems like this was done on purpose see 
https://github.com/pytorch/pytorch/pull/95653#discussion_r1119627765

Comment (mikaylagawarecki) [reply torch/nn/modules/activation.py:889]: Cancelling the merge since this was done purposefully

Comment (apivovarov) [reply torch/nn/modules/activation.py]: I reverted python code fixes and left only typos fixes

**Review by Skylion007 — APPROVED** [review]

**Pull request closed without merging**